docs/own-instance.rst

```diff
@@ -2,62 +2,62 @@
+To gain more insight on how these instances work, let's dive into how SearXNG
 protects its users.
 
 .. _SearXNG protect privacy:
 
 How does SearXNG protect privacy?
 =================================
 
-SearXNG protects the privacy of its users in multiple ways regardless of the type
-of the instance (private, public).  Removal of private data from search requests
+SearXNG protects the privacy of its users in multiple ways, regardless of the type
+of the instance (private or public).  Removal of private data from search requests
 comes in three forms:
 
- 1. removal of private data from requests going to search services
- 2. not forwarding anything from a third party services through search services
+ 1. Removing private data from requests going to search services
+ 2. Not forwarding anything from third party services through search services
     (e.g. advertisement)
- 3. removal of private data from requests going to the result pages
+ 3. Removing private data from requests going to the results pages
 
 Removing private data means not sending cookies to external search engines and
 generating a random browser profile for every request.  Thus, it does not matter
 if a public or private instance handles the request, because it is anonymized in
-both cases.  IP addresses will be the IP of the instance.  But SearXNG can be
+both cases.  The IP address used will be the IP of the instance, but SearXNG can also be
 configured to use proxy or Tor.  `Result proxy
 <https://github.com/asciimoo/morty>`__ is supported, too.
 
-SearXNG does not serve ads or tracking content unlike most search services.  So
+SearXNG does not serve ads or tracking content, unlike most search services.  Therefore,
 private data is not forwarded to third parties who might monetize it.  Besides
-protecting users from search services, both referring page and search query are
-hidden from visited result pages.
+protecting users from search services, both the referring page and search query are
+hidden from the results pages being visited.
 
 
 What are the consequences of using public instances?
 ----------------------------------------------------
 
 If someone uses a public instance, they have to trust the administrator of that
 instance.  This means that the user of the public instance does not know whether
-their requests are logged, aggregated and sent or sold to a third party.
+their requests are logged, aggregated, and sent or sold to a third party.
 
-Also, public instances without proper protection are more vulnerable to abusing
-the search service, In this case the external service in exchange returns
-CAPTCHAs or bans the IP of the instance.  Thus, search requests return less
+Also, public instances without proper protection are more vulnerable to abuse of
+the search service which may cause the external service to enforce
+CAPTCHAs, or to ban the IP address of the instance.  Thus, search requests would return less
```

Comment:
> "to enforce CAPTCHAs, or to ban the IP address"

is the additional comma before the “or” necessary?

Reply:
I think you're right in this case actually. The comma before "or" can be omitted, but we should add one between "service" and "which" as it's a "non-restrictive" clause. I'll see if I can edit the PR.